fellow-submissions/stephanie-smith.md

```diff
@@ -0,0 +1,108 @@
+
+**4. What does the following code log? Why?**
+    ```javascript
+    console.log(a);
+
+    var a = 1;
+    ```
+  - It will log undefined because `a` was declared, but not initialized. It then calls upon `a`, which is still undefined, and only after it's called is it defined to be equal to 1.
+  
+**5. What does the following code log? Why?**
+    ```javascript
+    console.log(a);
+
+    let a = 1;
+    ```
+  - This will log an error because `a` is not defined. `let` variables do not get initialized with anything, so there will be a reference error.
+
+**6. In the code snippet below, the function declaration comes _after_ the variable declaration in the code below. Why does it still work?**
+    ```javascript
+    let name = "Devonte";
+
+    function sayItLoud() {
+      console.log(`${name}!!!`)
+    }
+
+    sayItLoud();
+    ```
+  - Local scopes can access variables that are declared and initialized in the global scope, but global scopes can't access variables in the local scope, so in this case, it will still work.
+    
+**7. Why does the following code throw an error? What is the key difference between it and the example above?**
+    ```javascript
+    sayItLoud();
+
+    function sayItLoud() {
+      console.log(`${greeting}!!!}`)
+    }
+
+    let greeting = "Hello"
+    ```
+  - The function is called before it is even declared, so `greeting` doesn't even exist yet despite it being in the global scope.
+
+**8. What does the following code log? Why?**
+    ```javascript
+    let problemSet;
+
+    function complete(assignment) {
+      assignment = "done";
+    }
+
+    complete(problemSet);
+    console.log(`Status for Problem Set: ${problemSet}`);
+    ```
+  - It logs "Status for Problem Set: undefined" because problemSet was never initialized, so it becomes undefined.
+
+**9. Why does this code throw an error?** 
+    ```javascript
+    const isSunny🌞 = true;
+
+    if (isSunny🌞) {
+      let advice = 'Wear your sunshades!';
+    }
+
+    console.log(advice);
+    ```
+  - The variable `advice` is declared and initialized in the function, so it's not accessible in the global scope. Instead, it's better to call the function and return `advice` in the function.
+
+**10. So... why does this code work as intended?**
+    ```javascript
+    const isRainy🌧 = true;
+
+    if (isRainy🌧) {
+      var advice = 'Pack your umbrella.';
+    }
+
+    console.log(advice);
+    ```
+  - This code works because `var` is accessible regardless of scope. Even if `advice` is inside the function, the global scope can access it and log 'Pack your umbrella'.
```

Comment:
not true. For example:
```javascript
function testFunc() {
  var a = 15;
}

console.log(a); // ReferenceError
```

`var` variables have a scope but _how_ are those scopes defined?